src/utils/effortColors.js

```diff
@@ -1,64 +1,64 @@
-import _ from 'lodash';
+import { inRange } from 'lodash';
 
 //For progress bar that shows the percentage of the completion
 //Set 'invert' to true when used for the bars of TASK
-export const getProgressColor = (effort,commit,invert=false) => {
+export const getProgressColor = (effort, commit, invert = false) => {
   let color = 'white';
   let percentage = 0;
-  if (commit>0){
-    percentage = Math.round(effort*100/commit);
+  if (commit > 0) {
```

Comment:
Nice job on fixing the space. Thank you for your keen eye.

Reply:
Thanks for reviewing. FYI, if we have prettier configured, it auto corrects code spacing on file save.